### PR TITLE
Fix/nopreview

### PR DESCRIPTION
**Pull request by jkt001**
resized no_preview.png, added no_review sized 150x150 px, modify views and base_list.html accordingly

### CODE REVIEW

**Comment by jkt001** [comment]
this fixes issue #51 point 3

File: multimedia/views.py [hunk 1]
```diff
@@ -4,6 +4,8 @@
 )
 from crew.models import Crew
 
+no_preview_150 = '/statics/images/no-preview-150.png'
+no_preview = '/statics/images/no-preview.png'
```
Comment (digawp) [inline multimedia/views.py]: Can try `{% static \"images/no-preview.png\" %}` here for me? I think this is the better way, if possible

Comment (9gix) [inline multimedia/views.py]: :+1: as long as we don't hard code the URL.
For Static reverse outside of templates, we can still make use the templatetags at staticfiles.
`django.contrib.staticfiles.templatetags.staticfiles.static`

```
from django.contrib.staticfiles.templatetags.staticfiles import static
no_preview = static('images/no-preview.png')
```

**Comment by jkt001** [comment]
rebased this branch to master and update the image reference to static

**Comment by jkt001** [comment]
somehow after I rebased, changes that I did not make appear in the "files changed" tab. Don't know why, but this is ready for merge.

**Comment by digawp** [comment]
Shouldn't have rebased in this case. I guess @9gix is right, just merge unless you are sure what you are doing. I am not sure what will happen if we merge this, bcuz usually if this happened to me during 2103, I will fix the branch so that changes from the master does not appear in the PR.
@9gix pls advise?

**Pull request merged**